src/test/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulationTest.java

```diff
@@ -0,0 +1,48 @@
+package com.williamfiset.algorithms.geometry;
+
+import java.awt.geom.*;
+import org.junit.*;
+
+public class MinimumCostPolygonTriangulationTest {
+
+  @Test
+  public void MinimumCostPolygonTriangulationBasicTest() {
+    Point2D[] pts = new Point2D[5];
+
+    pts[0] = new Point2D.Double(0, 0);
+    pts[1] = new Point2D.Double(1, 0);
+    pts[2] = new Point2D.Double(2, 1);
+    pts[3] = new Point2D.Double(1, 2);
+    pts[4] = new Point2D.Double(0, 2);
+
+    double cost = MinimumCostPolygonTriangulation.minimumCostTriangulation(pts);
+    assert (Math.abs(cost - 15.3) < .001);
```

Comment:
Do you want to try using assertThat from the Google Truth framework to check relative tolerance? I would also define the tolerance as a static final constant in this file.

import static com.google.common.truth.Truth.assertThat;
https://truth.dev/floating_point.html

So, something like:
assertThat(value).usingTolerance(1.0e-10).isEqualTo(otherValue);